test/lit/passes/instrument-memory-filter.wast

```diff
@@ -0,0 +1,344 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+;; NOTE: This test was ported using port_passes_tests_to_lit.py and could be cleaned up.
+
+;; RUN: foreach %s %t wasm-opt --instrument-memory="i32.load,memory.grow" -S -o - | filecheck %s
+
```

Comment:
Please add a comment explaining that only `i32.load` and `memory.grow` should be instrumented below.

Reply:
I don't see the comment - where is it?

The first function should have a comment "we instrument grow and loads because we were asked to by the flags. The second, "we do not instrument store since we were not asked to by the flags".

Reply:
Ah, I see now - I was looking at the top of the file, where I commented, expecting to see the new comment there. I see the ones down below, and they look great.